.github/workflows/docker-image.yml

```diff
@@ -1,112 +1,200 @@
 name: '[Docker] Build and Publish Runner Images'
 
 on:
- pull_request:
-   paths:
-     - "images/ubuntu/**"
- push:
-   branches: [main]
-   paths:
-     - ".github/workflows/docker-image.yml"
-     - "images/ubuntu/**"
- workflow_dispatch:
-   inputs:
-     release_type:
-       description: 'Release type (devel/latest)'
-       required: true
-       default: 'devel'
+  pull_request:
+    paths:
+      - .github/workflows/**
```

Comment:
Could potentially just target this one workflow

Reply:
tagging @sandipanpanda for viz (so we can tidy this stuff up a bit more)